### PR TITLE
Feature/legacy data

### PR DESCRIPTION
**Pull request by Adamws33**
updated selector to use legacy data API for 
Sidebar categories
Sidebar filters
Sources
data-dictionary

### CODE REVIEW

**Comment by Adamws33** [comment]
> 

Done!

**Comment by mikahpinegar** [comment]
comparison view in cover crop list is broken

**Comment by mikahpinegar** [comment]
when going to cc list from explorer it shows no goals in comparison view, when coming from selector it only shows the goals selected by the user

**Comment by mikahpinegar** [comment]
clicking on a crop to open info sheet makes approx 50 api calls to `/data-dictionary`

**Comment by mikahpinegar** [comment]
the data dictionary should probably only be fetched once instead of in a bunch of places

**Comment by Adamws33** [comment]
> lot's of crops missing from `/data-dictionary` ex: barley winter, brassica forage in zone 6

This will have to be updated on the back end.  It sounds like there are things missing from the data in the database.

**Comment by Adamws33** [comment]
> the data dictionary should probably only be fetched once instead of in a bunch of places

i can implement that now but i thought since we were changing things soon to be more dynamic we were just focusing on getting it converted over from the local data to the API.  

same for the 50 calls on the infosheet call.  I noticed that im pretty sure its doing one call per crop to find the right one.  i figured it was best just to get this API working like it was before since its getting redone soon.

File: src/pages/CropSidebar/CropSidebar.js [hunk 1]
```diff
@@ -63,6 +63,31 @@ const CropSidebar = ({
...
+        .then((data) => { setSidebarCategoriesData(data); })
+        .catch((err) => {
+          // eslint-disable-next-line no-console
+          console.log(err.message);
+        });
+    }
+
+    getFilterData();
+    getCategoriesData();
+  }, []);
```
Comment (mikahpinegar) [inline src/pages/CropSidebar/CropSidebar.js:89]: this should trigger every time zone changes right?

**Review by mikahpinegar — CHANGES_REQUESTED** [review]

**Review by mikahpinegar — APPROVED** [review]

**Pull request merged**